docs/section12/Rule12-1.md

```diff
@@ -1,36 +1,37 @@
-
-- Get the total number of spaces in the building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
-
-  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```
-
-  - For each space in zone, add to the total number of spaces in the Baseline model: ```for space_baseline in zone_baseline.spaces: num_of_spaces_baseline += 1```  
-
-**Rule Assertion:** ```num_of_spaces_user == num_of_spaces_baseline```  
-
-**[Back](../_toc.md)**
+**Evaluation Context:** Each RPD  
+
+**Applicability Checks:**  
+N/A
+
+**Function Call:**  
+match_data_element
+
+## Rule Logic:
+- Store a list of any miscellaneous equipment power where proposed is greater than baseline: `unexpected_misc_equipment_power = []`
+- Store a list of any miscellaneous equipment power where proposed is less than baseline: `reduced_misc_equipment_power = []`
+- For each miscellaneous equipment load in the baseline RMD: `for misc_equipment_b in B_RMD...miscellaneous_equipment:`
+  - Get the corresponding miscellaneous equipment load in the proposed RMD: `misc_equipment_p = match_data_element(P_RMD, MiscellaneousEquipment, misc_equipment_b.id)`
+  - Get the miscellaneous equipment power in the proposed RMD: `misc_equipment_power_p = misc_equipment_p.power`
+  - Get the miscellaneous equipment power in the baseline RMD: `misc_equipment_power_b = misc_equipment_b.power`
+  - If the miscellaneous equipment power in the baseline RMD is greater than the miscellaneous equipment power in the proposed RMD: `if misc_equipment_power_b > misc_equipment_power_p:`
+    - Append the miscellaneous equipment info for reporting: `reduced_misc_equipment_power.append({"id": misc_equipment_b.id, "baseline_power": misc_equipment_power_b, "proposed_power": misc_equipment_power_p})`
+  - If the miscellaneous equipment power in the baseline RMD is less than the miscellaneous equipment power in the proposed RMD: `elif misc_equipment_power_b < misc_equipment_power_p:`
+    - Append the miscellaneous equipment info for reporting: `unexpected_misc_equipment_power.append({"id": misc_equipment_b.id, "baseline_power": misc_equipment_power_b, "proposed_power": misc_equipment_power_p})`
+**Rule Assertion:**  
+  - Case 1: If all equipment power was modeled identically: PASS `if len(unexpected_misc_equipment_power) == 0 and len(reduced_misc_equipment_power) == 0: PASS`
+  - Case 2: Else if any equipment power is reduced in the proposed: UNDETERMINED: `elif len(unexpected_misc_equipment_power) == 0: UNDETERMINED and raise_message="The proposed building miscellaneous equipment load is less than the baseline, which is only permitted when the model is being used to quantify performance that exceeds the requirements of Standard 90.1."`
```

Comment:
This could also incorporate RPD.compliance_path. Based on the labeling it appears that whether it is for minimum code compliance or beyond code performance would be indicated by the data element. If it says minimum code compliance that could have a fail outcome.

Reply:
Good point!